mappings/net/minecraft/client/option/GameOptions.mapping

```diff
@@ -45,50 +75,78 @@ CLASS net/minecraft/class_315 net/minecraft/client/option/GameOptions
 	FIELD field_1865 invertYMouse Z
 	FIELD field_1866 debugEnabled Z
 	FIELD field_1867 keySprint Lnet/minecraft/class_304;
+		COMMENT A key binding for sprinting.
+		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_KEY_LEFT_CONTROL the left control key} by default.
 	FIELD field_1868 guiScale I
 	FIELD field_1869 keyDrop Lnet/minecraft/class_304;
+		COMMENT A key binding for dropping the selected item.
+		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_KEY_Q the Q key} by default.
```

Comment:
```suggestion
		COMMENT A key binding for dropping the item in the selected slot.
		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_KEY_Q the Q key} by default.
		COMMENT
		COMMENT <p>The selected slot is the slot the mouse is over when in a screen.
		COMMENT Otherwise, it is the main hand.
```
See https://github.com/FabricMC/yarn/pull/2640#discussion_r688859388.